README.md

```diff
@@ -0,0 +1,51 @@
+<br />
+<p align="center">
+  <h3 align="center">Attendance as a Service</h3>
+
+  <p align="center">
+    Microservice for signing up users for events
+    <br />
+    <a href="https://github.com/dotkom/a3s/issues">Report Bug</a>
+    |
+    <a href="https://github.com/dotkom/a3s">Request Feature</a>
+  </p>
+</p>
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+- [Development](#development)
+
+## Getting Started
+
+The service is a backend GraphQL API which is written in Golang. It uses [gqlgen](gqlgen) to create strongly typed
+GraphQL APIs and [Entgo](entgo) as a ORM in front of a PostGreSQL database.
```

Comment:
These footnote links do not work in the preview. Will they be fixed when pushing to `main`?

Reply:
Wooops the links were broken, fixed!